src/zcbor_decode.c

```diff
@@ -606,7 +606,7 @@ bool zcbor_any_decode(zcbor_state_t *state, void *result)
 			payload_bak = state->payload;
 			state->elem_count = value;
 			if (!zcbor_multi_decode(value, value, &num_decode,
-					(void *)zcbor_any_decode, state,
+					(zcbor_decoder_t *)zcbor_any_decode, state,
```

Comment:
Why don't we need `struct` here?

Reply:
`zcbor_decoder_t` is a function pointer type.